x-pack/test/cloud_security_posture_functional/pages/rules.ts

```diff
@@ -181,5 +186,78 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         expect((await rule.rulePage.getEnableSwitchButtonState()) === 'true').to.be(true);
       });
     });
+
+    describe('Rules Page - Rules Counters', () => {
```

Comment:
@JordanSh @animehart mind that the whole suit is skipped (see line 31) so it might conflict with https://github.com/elastic/kibana/pull/176029/files . We need to make sure if this new set of tests is green when the main `describe` is unskipped